### PR TITLE
chore: Fix nightlies after recent changes

### PR DESCRIPTION
**Pull request by tgodzik**
The issues caused by https://github.com/lampepfl/dotty/pull/15877

### CODE REVIEW

File: mtags/src/main/scala-3/scala/meta/internal/pc/MetalsInteractive.scala [hunk 1]
```diff
@@ -259,6 +259,9 @@ object MetalsInteractive:
               indexed.ctx.source,
             )
           then List((head.symbol, head.typeOpt))
+          // https://github.com/lampepfl/dotty/issues/15937
+          else if head.isInstanceOf[TypeTree] then
+            enclosingSymbolsWithExpressionType(path.tail, pos, indexed)
```
[the remark this answers is not in the document]
Comment (kpodsiad) [reply mtags/src/main/scala-3/scala/meta/internal/pc/MetalsInteractive.scala:264]: or even rename `tl` to `tail` as it's much common name for tail ;)

[the remark this answers is not in the document]
Comment (tgodzik) [reply mtags/src/main/scala-3/scala/meta/internal/pc/MetalsInteractive.scala:264]: This is actually why I haven't found it 😂

File: mtags/src/main/scala-3/scala/meta/internal/pc/MetalsInteractive.scala [hunk 2]
```diff
@@ -259,6 +259,9 @@ object MetalsInteractive:
               indexed.ctx.source,
             )
           then List((head.symbol, head.typeOpt))
+          // https://github.com/lampepfl/dotty/issues/15937
```
Comment (tanishiking) [inline mtags/src/main/scala-3/scala/meta/internal/pc/MetalsInteractive.scala:262]: Can you elaborate on which case applies this pattern?

**Review by tanishiking — APPROVED** [review]
LGTM, I just left one question!

File: mtags/src/main/scala-3/scala/meta/internal/pc/MetalsInteractive.scala (hunk 1 above)
[the remark this answers is not in the document]
Comment (tgodzik) [reply mtags/src/main/scala-3/scala/meta/internal/pc/MetalsInteractive.scala]: Fixed and renamed.

File: mtags/src/main/scala-3/scala/meta/internal/pc/MetalsInteractive.scala (hunk 2 above)
Comment (tgodzik) [reply mtags/src/main/scala-3/scala/meta/internal/pc/MetalsInteractive.scala]: Added a comment!

**Review by tanishiking — APPROVED** [review]
LGTM!

**Pull request merged**